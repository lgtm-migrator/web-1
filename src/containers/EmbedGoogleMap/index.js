import React from "react"
import { EmbedGoogleMap as MuyEmbedGoogleMap } from "muy"

const EmbedGoogleMap = ({ coordinates, ...props }) => (
  <MuyEmbedGoogleMap
    apiKey={process.env.â€©GATSBY_GOOGLE_MAPS_EMBED_API_KEY}
    coordinates={coordinates}
    {...props}
  />
)

export default EmbedGoogleMap
