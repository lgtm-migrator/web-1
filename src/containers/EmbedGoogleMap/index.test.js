import React from "react"
import renderer from "react-test-renderer"
import EmbedGoogleMap from "."

describe("<EmbedGoogleMap /> container", () => {
  it("should render", () => {
    const tree = renderer.create(<EmbedGoogleMap />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
