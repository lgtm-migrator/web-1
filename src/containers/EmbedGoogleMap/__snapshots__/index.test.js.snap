// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmbedGoogleMap /> container should render 1`] = `
<iframe
  allowFullScreen={true}
  className="MuiBox-root MuiBox-root-3"
  frameBorder={0}
  src="https://www.google.com/maps/embed/v1/place?q=undefined&key=undefined"
/>
`;
